{
  "invalid_api_params": {
    "http_code": "400",
    "code": "BAD_REQUEST",
    "message": "At least one parameter is invalid or missing. See err.error_data array for more details."
  },
  "exception": {
    "http_code": "500",
    "code": "INTERNAL_SERVER_ERROR",
    "message": "Something went wrong exception."
  },
  "ddb_method_call_error":{
    "http_code": "422",
    "code": "UNPROCESSABLE_ENTITY",
    "message": "DynamoDB method was called successfully but returned error"
  },
  "auto_scale_method_call_error":{
    "http_code": "422",
    "code": "UNPROCESSABLE_ENTITY",
    "message": "Auto Scale method was called successfully but returned error"
  }
}